### ISSUE TITLE
NextJS: TS2322 KeyboardDatePicker 

### ISSUE THREAD

**Issue by kodermax** [mui-org/material-ui-pickers#1214]
![A GIF or MEME to give some spice of the internet](url)

## Environment

| Tech                 | Version |
| -------------------- | ------- |
| @material-ui/pickers |    3.2.0     |
| material-ui          |    4.3.0     |
| React                |     16.8.6    |
| Browser              |    Chrome     |

## Steps to reproduce
When I use KeyboardDatePicker with NextJS i get error.
![image](https://user-images.githubusercontent.com/590985/62157070-0456e700-b315-11e9-9c98-4ba12dc86eb6.png)

```
return (
    <ThemeProvider theme={defaultMaterialTheme}>
      <MuiPickersUtilsProvider utils={DateFnsUtils}>
        <KeyboardDatePicker
          disableFuture
          placeholder="MM/DD/YYYY"
          openTo="year"
          name={props.name}
          label={props.label}
          format="MM/dd/yyyy"
          value={props.value}
          onChange={props.onChange}
          views={['year', 'month', 'date']}
          InputProps={{ classes: inputClasses, disableUnderline: false }}
          TextFieldComponent={CssTextField}
          {...props}
        />
      </MuiPickersUtilsProvider>
    </ThemeProvider>
  )```

**Comment by dmtrKovalenko** [comment]
This looks like typescript version conflict between `core` and `us `

**Comment by dmtrKovalenko** [comment]
Should also be fixed by #1223
